dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -305,7 +305,7 @@ connectionHarvestTriggerCount.desc=JDBC driver property: connectionHarvestTrigge
 connectionPoolName=Connection pool name
 connectionPoolName.desc=JDBC driver property: connectionPoolName.
 connectionProperties=Connection properties
-connectionProperties.desc=JDBC driver property: connectionProperties.
+connectionProperties.desc=JDBC driver property: connectionProperties.  The ssl properties javax.net.ssl.keyStorePassword and javax.net.ssl.trustStorePassword may be encrypted using the same process used for other datasource passwords.
```

Comment:
"may be encrypted" is confusing because it isn't clear if this means that Liberty (or the Oracle driver) might choose to do some encrypting with these values or if it means we are allowing for the user to specify encrypted values.
How about re-wording as:
"You can configure encrypted values for the ssl properties javax.net.ssl.keyStorePassword and javax.net.ssl.trustStorePassword using the same process used for other data source passwords."
?